pennylane/templates/layers/basic_entangler.py

```diff
@@ -14,43 +14,14 @@
 r"""
 Contains the ``BasicEntanglerLayers`` template.
 """
-# pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+import numpy as np
```

Comment:
Do we not need the PL version of numpy? It looks like we now need to import numpy for the parameter initialization, which I guess should be wrapped as we might want to differentiate wrt them?

Reply:
We could, but I thought the user has to convert tf and torch anyways, so rather return vanilla numpy arrays...but we may solve this differently now anyways (see above)